app/src/styles.sass

```diff
@@ -142,18 +138,24 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
 @include angular-material-typography($itero-mat-typography)
 
 
-.sidemenu
-  border-left:1px solid #704f6a
-  border-top: 1px solid #704f6a
-  border-bottom: 1px solid #704f6a
+
+.info-panel
+  border: 1px solid #704f6a
+  border-right: none
   background-color: #d5b6d0
   float: right
-  height: 100%
-  margin-right: -16px
   margin-left: 3px
-  box-shadow: -2px 0px 2px 0 rgba(255, 204, 244, 0.9)
-  .helpbutton
-    text-align: center
+  box-shadow: -5px 0px 5px 0 rgba(255, 204, 244, 0.7)
+  width: 80%
+  margin-right: 0
+  position: absolute
+  right: 0
+  display: flex
+  flex-flow: row nowrap
+  z-index: 1
+  .info-panel-icon 
+    #help-button
+      margin: 17px 3px
```

Comment:
I understand that you want this margin to be the same as the margin of the p element in div.info-panel-info. But you cannot rely on the fact that this margin is approximatively 17px, because
1. the margin of the p element is expressed in ex (or em) unit, which means that it depend on the font used;
2. this is a default property of p, meaning that in may depend on the browser.

For the horizontal margin, maybe could you use a variable from size.sass?